backend/README.md

```diff
@@ -50,9 +50,34 @@ In order for the Tesseract exe to be found in your python files, we need to eith
 pytesseract.pytesseract.tesseract_cmd = [path/to/Tesseract-OCR/tesseract.exe]
 ```
 
-E.g.
+## Running the API
+
+While your virtual environment is activated, run the following command:
+
+```
+python main.py
+```
+
+This will run the API programatically, and allow you to interact with it.
+
+You can view the available endpoints and test the API via the /docs (Swagger UI) endpoint:
+
+> `http://localhost:5000/docs`
+
+### Sending a picture
+
+To send a picture to the API via alternative methods other than the Swagger UI, we need to setup a few things.
+
+The header for the request should contain the following:
+
+```
+{
+    "accept": "application/json",
+    "Content-Type":"multipart/form-data"
+}
+```
 
-> pytesseract.pytesseract.tesseract_cmd = r"C:\Users\YOUR_NAME\AppData\Local\Tesseract-OCR\tesseract.exe"
+For the request content, we need to use the multipart datatype, with a single `file` (jpeg) attached.
```

Comment:
Maybe we could add a section for the Swagger doc and then reference that here

Reply:
yes, thats much simpler. Will add

Reply:
Thank you thank you :)